b2share/modules/records/b2share_epic.py

```diff
@@ -2,18 +2,123 @@
 from simplejson import dumps as jsondumps
 from werkzeug.exceptions import abort
 from flask import current_app
+from datetime import datetime
 from urllib.parse import urljoin, urlparse
+from b2handle.handleclient import EUDATHandleClient
 
 from .errors import EpicPIDError
 
 
-def createHandle(location, checksum=None, suffix=''):
+handle_client = None
+handle_prefix = None
+
+
+def init_handle_client(app):
```

Comment:
Why not do the initialization in an extension as we do for other modules?
We could have `b2share.module.handle` with a `proxy.py` providing `current_handle`.

Reply:
Can you make this change? I don't know what are the advantages of the Invenio way, and just used known Python features.